src/Components/Components/src/SupplyParameterFromQueryAttribute.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Components
```

Comment:
Would this go in the routing namespace since it's closely related to it?

Reply:
Honestly I'm meh either way. I put it here to be alongside its close friend, `ParameterAttribute`. But if you have even the slightest preference for moving it, I'm happy to do so.

Reply:
I think this is fine where it is and advocate against it going inside routing. Unless we want to change the definition of routing for Blazor Webassembly, the query string parameters are not involved in selecting a handler and for that reason they aren't part of routing.